### PR TITLE
Use msgs6/fuel-tools5/physics3/gazebo4/sdf10 in ignition-dome

### PR DESCRIPTION
**Pull request by j-rivero**
We need to create the branches first. Not sure if we want prereleases too.

### CODE REVIEW

**Comment by chapulina** [comment]
No need for pre-releases yet, I think we can go on with `master` like we're doing for `ign-gazebo`, `ign-rendering`, etc.

I think we should also update all individual libraries to stay in sync, not just `collection-dome`. So this needs to be updated for example:

https://github.com/ignition-tooling/gazebodistro/blob/b7d693226e13b8b83be9008e6be6a21256c5e29d/ign-gazebo4.yaml#L26-L29

**Comment by j-rivero** [comment]
> No need for pre-releases yet, I think we can go on with `master` like we're doing for `ign-gazebo`, `ign-rendering`, etc.
> 
> I think we should also update all individual libraries to stay in sync, not just `collection-dome`. So this needs to be updated for example:

thanks, updated.

**Comment by j-rivero** [comment]
> It all looks right to me. But I think we need the bumps in the source code before merging.

Both ign-msgs and ign-fuel-tools have the master branch bumped to 6 and 5 major versions. Do we need to do anything more there?

**Comment by chapulina** [comment]
Oh sorry, I meant the downstream packages' `find_package` calls need to be updated to find these versions.

---

Here's a list of everything I think needs to be done when bumping a library's version for the upcoming collection:

- [ ] Update `gazebodistro` like you're doing here
- [ ] Update `release-tools` to build the `master` branch from source for downstream libs (like [this](https://github.com/ignition-tooling/release-tools/blob/93093d3d1f50e006458b2ea2edc13c20cc676577/jenkins-scripts/docker/ign_sensors-compilation.bash#L31-L38))
- [ ] Update `homebrew-simulation` formulas and aliases like https://github.com/osrf/homebrew-simulation/pull/801/files
- [ ] Update all downstream packages' `find_package` calls to get the new version

Since we need to bump all of these for Dome, that's going to be a lot of PRs to a lot of downstream dependencies all the way up to `ign-launch`. So I think we may want to break them into chunks.

* `sdformat10`
* `ign-physics3`
* `ign-fuel-tools5`
* `ign-msgs6`
* `ign-transport9`

**Comment by j-rivero** [comment]
thanks for the summary! I start to work on the different points.

**Comment by scpeters** [comment]
I updated just one file in 2de7bcb, but the correct branch for sdformat10 is `sdf10`; `master` is already receiving changes for `libsdformat11`

**Comment by j-rivero** [comment]
> I updated just one file in [2de7bcb](https://github.com/ignition-tooling/gazebodistro/commit/2de7bcb32c0d9f967db0d8179f99a9b8c34699c3), but the correct branch for sdformat10 is `sdf10`; `master` is already receiving changes for `libsdformat11`

thanks!

File: collection-dome.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ repositories:
   ign-transport:
     type: git
     url: https://github.com/ignitionrobotics/ign-transport
-    version: ign-transport8
+    version: master
   sdformat:
     type: git
     url: https://github.com/osrf/sdformat
```
Comment (chapulina) [inline collection-dome.yaml:60]: This needs to use `sdf10`

File: ign-gazebo4.yaml [hunk 2]
```diff
@@ -50,8 +50,8 @@ repositories:
   ign-transport:
     type: git
     url: https://github.com/ignitionrobotics/ign-transport
-    version: ign-transport8
+    version: master
   sdformat:
     type: git
     url: https://github.com/osrf/sdformat
-    version: sdf9
+    version: master
```
Comment (chapulina) [inline ign-gazebo4.yaml:57]: SDF development is a bit different from the rest now. We have the `sdf10` branch for Dome and `master` will be SDF 11, for Ignition-E

File: ign-gui4.yaml [hunk 3]
```diff
@@ -18,7 +18,7 @@ repositories:
   ign-msgs:
     type: git
     url: https://github.com/ignitionrobotics/ign-msgs
-    version: ign-msgs5
+    version: ign-msgs6
```
Comment (chapulina) [inline ign-gui4.yaml:21]:
```suggestion
    version: master
```

And also need to update transport

File: ign-launch3.yaml [hunk 4]
```diff
@@ -54,8 +54,8 @@ repositories:
   ign-transport:
     type: git
     url: https://github.com/ignitionrobotics/ign-transport
-    version: ign-transport8
+    version: master
   sdformat:
     type: git
     url: https://github.com/osrf/sdformat
-    version: sdf9
+    version: master
```
Comment (chapulina) [inline ign-launch3.yaml:61]:
```suggestion
    version: sdf10
```

File: ign-launch3.yaml [hunk 5]
```diff
@@ -30,7 +30,7 @@ repositories:
   ign-msgs:
     type: git
     url: https://github.com/ignitionrobotics/ign-msgs
-    version: ign-msgs5
+    version: master
   ign-physics:
     type: git
     url: https://github.com/ignitionrobotics/ign-physics
```
Comment (chapulina) [inline ign-launch3.yaml:36]: physics needs `master`

File: collection-dome.yaml [hunk 6]
```diff
@@ -30,7 +30,7 @@ repositories:
   ign-msgs:
     type: git
     url: https://github.com/ignitionrobotics/ign-msgs
-    version: ign-msgs5
+    version: master
   ign-physics:
     type: git
     url: https://github.com/ignitionrobotics/ign-physics
```
Comment (chapulina) [inline collection-dome.yaml:36]: physics needs master

**Pull request merged**